### DISCUSSION TITLE
Report: Northgard

### DISCUSSION THREAD

**Discussion by cfrenette** [canonical/steam-snap#213] [Game Reports]
```
os_release: 
    name:               "Ubuntu"
    version:            "22.04.2 LTS (Jammy Jellyfish)"
snap_info: 
    steam_revision:     136
    snapd_revision:     18933
lspci: 
    00:02.0:            Intel Corporation TigerLake-LP GT2 [Iris Xe Graphics] (rev 01)
glxinfo: 
    gpu:                Mesa Intel(R) Xe Graphics (TGL GT2)
    gpu_version:        4.6 (Core Profile) Mesa 23.0.2 - kisak-mesa PPA
lscpu: 
    model_name:         11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz
xdg_current_desktop:    ubuntu:GNOME
desktop_session:        ubuntu
```

fails to launch (no error received)

**Comment by UnrealisticDino** [comment]
With proton it worked for me and I had the same issue with native but was able to get a log. Looks like native Dead Cells has the same issue.

```
os_release: 
    name:               "Ubuntu"
    version:            "22.04.2 LTS (Jammy Jellyfish)"
snap_info: 
    steam_revision:     137
    snapd_revision:     18933
lspci: 
    08:00.0:            NVIDIA Corporation GP104 [GeForce GTX 1070] (rev a1)
glxinfo: 
    gpu:                NVIDIA GeForce GTX 1070/PCIe/SSE2
    gpu_version:        4.6.0 NVIDIA 530.41.03
lscpu: 
    model_name:         AMD Ryzen 7 3800X 8-Core Processor
xdg_current_desktop:    ubuntu:GNOME
desktop_session:        ubuntu
```


ERROR: ld.so: object '/home/juan/snap/steam/common/.local/share/Steam/ubuntu12_64/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS64): ignored.
ERROR: ld.so: object '/home/juan/snap/steam/common/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
ERROR: ld.so: object '/home/juan/snap/steam/common/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.
** ERROR ** Missing library libXss.so.1
** ERROR ** Missing library libpulsecommon-15.99.so
** ERROR ** Missing library libXss.so.1
** ERROR ** Missing library libpulsecommon-15.99.so
** ERROR ** Missing library libXss.so.1
** ERROR ** Missing library libpulsecommon-15.99.so
** ERROR ** Missing library libXss.so.1
** ERROR ** Missing library libpulsecommon-15.99.so
** ERROR ** Missing library libXss.so.1
** ERROR ** Missing library libpulsecommon-15.99.so
sh: 1: notify-send: not found
src/module.c(373) : FATAL ERROR : Failed to load library sdl.hdll
